src/olympia/devhub/forms.py

```diff
@@ -508,8 +508,8 @@ def __init__(self, *args, **kw):
         self.version = kw.pop('version', None)
         super(NewUploadForm, self).__init__(*args, **kw)
 
-        # If we have a version, pre-select the ones from the previous
-        # version to help the developer decide.
+        # If we have a version, pre-select the ones from the current version
+        # to help the developer decide.
         if self.version:
```

Comment:
You can drop everything `self.version` now, right?  Because you can't have any new uploads for a version once it exists.

Reply:
True, I think I was too cautious. I'll get rid of all that.